python/paddle/tensor/__init__.py

```diff
@@ -191,6 +191,8 @@
 from .math import diagonal  # noqa: F401
 from .math import rad2deg  # noqa: F401
 from .math import deg2rad  # noqa: F401
+from .math import gcd  # noqa: F401
+from .math import lcm  # noqa: F401
```

Comment:
shall we add gcd and lcm in tensor_method_func  below ?

Reply:
done